tests/cnf/core/network/security/internal/tsparams/securityvars.go

```diff
@@ -0,0 +1,21 @@
+package tsparams
+
+import (
+	"github.com/openshift-kni/eco-gotests/tests/cnf/core/network/internal/netparam"
+	"github.com/openshift-kni/k8sreporter"
+)
+
+var (
+	// Labels represents the range of labels that can be used for test cases selection.
+	Labels = append(netparam.Labels, LabelSuite)
+
+	// ReporterNamespacesToDump tells to the reporter from where to collect logs.
+	ReporterNamespacesToDump = map[string]string{
+		TestNamespaceName: TestNamespaceName,
+	}
+
+	// TestNamespaceName security-tests namespace where all test cases are performed.
+	TestNamespaceName = "security-tests"
+	// ReporterCRDsToDump tells to the reporter what CRs to dump.
+	ReporterCRDsToDump = []k8sreporter.CRData{}
```

Comment:
Please add CRs what you want to dump after the failure. machineconfigs or something else

Reply:
Let me know when you are free for a call

Reply:
added